samples/Server/src/lib/chat/moderator.ts

```diff
@@ -18,7 +17,7 @@ export const createThread = async (topicName?: string): Promise<string> => {
   const chatClient = new ChatClient(getEndpoint(), credential);
 
   const request: CreateChatThreadRequest = {
-    topic: topicName ?? GUID_FOR_INITIAL_TOPIC_NAME
+    topic: 'Your Chat sample'
```

Comment:
this function takes in an optional `topicName` so keep the `topicName ?? `
```suggestion
    topic: topicName ?? 'Your Chat sample'
```

Reply:
Updated :)